lisa_commands/commands.py

```diff
@@ -252,14 +252,53 @@ def image_me(query, _slack_event):
         return "From Google: " + request.google_image_search(query)
     return "From Bing: " + request.bing_image_search(query)
 
-def kill_me(_query, _slack_event):
+def kill_me(_query, slack_event):
     """
     query - query str (unused for this function)
-    slack_event - A dict of slack event information (unused for this function)
+    slack_event - A dict of slack event information
 
-    Returns a link to a gif
+    Returns nothing
     """
-    return request.gify_search('gifs', 'kill me')
+
+    # kick user from channel
+    data = {
+        'channel': slack_event["channel"],
+        'user': slack_event['user'],
+    }
+
+    response = json.loads(request.submit_slack_request(data, 'channels.kick', 'USER'))
+    if response['ok']:
+        text = random.choice([
+            'By order of the SDF Armed Forces you have been killed',
+            'Ask not for whom the bell tolls. It tolls for thee ~ John Donne',
+            "We all have been one life to live. Well you do. I'm a bot, and I have backups",
+            'A coward dies a thousand times before thier death, but the valiant \n\
```

Comment:
Fun tidbit: To better handle multi-line strings, you can use the "string continuation format" like so!
```python
super_long_string = ('first part, blah blah blah... '
    'second long part, even longer, blah blah...'
    ' and so on!')
```
This format allows easy formatting of long strings. Notably, however, they will be treated as _one_ string, IE without newlines. If newlines are important, then \n use remains important.